test/test_unit_test_runner_test.rb

```diff
@@ -0,0 +1,101 @@
+# typed: true
+# frozen_string_literal: true
+
+require "test_helper"
+require "stringio"
+
+module RubyLsp
+  class TestUnitTestRunnerTest < Minitest::Test
+    def test_test_runner_output
+      stdout, _stderr, _status = Open3.capture3(
+        "bundle",
+        "exec",
+        "ruby",
+        "test/fixtures/test_unit_example.rb",
+        "--runner",
+        "ruby_lsp",
+      )
+      actual = parse_output(stdout)
+
+      project_path = File.expand_path("lib/ruby_lsp")
```

Comment:
Our fixture exists in `#{Dir.pwd}/test/fixtures` not in `#{Dir.pwd}/lib/ruby_lsp/test/fixtures` (which doesn't exist). I think this is a result of using `__dir__` as the base for expand path.